README.md

```diff
@@ -126,17 +126,17 @@ You can add a global option to every Kinvey CLI command to get the described beh
 
    When used after a `kinvey-cli` command, shows its usage instructions.
 
-* `--host <host>`
+* `--instanceId <instance ID>`
 
-   Hostname of a Kinvey instance. It has a default value of `https://manage.kinvey.com/` which most customers should use. If you are a customer on a dedicated Kinvey instance, enter your dedicated host name as either an absolute URI (`https://kvy-us2-manage.kinvey.com/`) or an instance name only (`kvy-us2`).
+   ID (e.g., `kvy-us2`) or full hostname (e.g., `https://kvy-us2-manage.kinvey.com/`) of a Kinvey instance. It has a default value of `kvy-us1` (or `https://manage.kinvey.com/`) which most customers should use. If you are a customer on a dedicated Kinvey instance, enter your dedicated instance ID.
 
```

Comment:
I don't think that clients need to supply full hostname as the instanceId is enough. I'd suggest that we provide info only about the instance ID and skip the hostname part.

Reply:
The reason I added this is to let upgrading users get used to the new way; to make it easier for them to understand that instance ID replaces the hostname; to allow a transition period. But I can remove it if required.

Reply:
It makes sense when you put it like that :satisfied: Your call.